Week1/homework/info_from_data.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password: 'hyfpassword',
+    database: 'world',
+});
+connection.connect(function(err) {
+    if (err) console.log(err);
+    else console.log(`Connected!to database: world`);
+});
+const infoFromData = [
+    'SELECT name FROM country WHERE population > 8000000 ',
+    'SELECT name FROM country WHERE name LIKE "%land%"',
+    'SELECT name FROM city WHERE population BETWEEN 500000 AND 1000000',
+    'SELECT name FROM country WHERE Continent = "Europe"',
+    'SELECT name FROM country ORDER BY SurfaceArea DESC;',
+    'SELECT city.name AS NederlandsCities FROM city INNER JOIN country ON city.CountryCode = country.Code WHERE country.name = "Netherlands"',
```

Comment:
It's good but INNER keyword is not necessary, JOIN is INNER by default .

Reply:
![Screenshot_20190911-215807_1](https://user-images.githubusercontent.com/50738429/64730511-ccc88680-d4df-11e9-982b-a31e8b70885f.png)
Is this explanation in the photo correct?  then all the words in ( ) are not necessarily, Like what you mentioned.

Reply:
thanks,
I post a link here as a reference for me. 
https://www.w3resource.com/mysql/advance-query-in-mysql/mysql-joins.php